lib/std/fs.zig

```diff
@@ -1285,15 +1285,6 @@ pub const Dir = struct {
             .SecurityDescriptor = null,
             .SecurityQualityOfService = null,
         };
-        if (sub_path_w[0] == '.' and sub_path_w[1] == 0) {
-            // Windows does not recognize this, but it does work with empty string.
-            nt_name.Length = 0;
-        }
-        if (sub_path_w[0] == '.' and sub_path_w[1] == '.' and sub_path_w[2] == 0) {
-            // If you're looking to contribute to zig and fix this, see here for an example of how to
-            // implement this: https://git.midipix.org/ntapi/tree/src/fs/ntapi_tt_open_physical_parent_directory.c
```

Comment:
I don't see this addressed in the PR description? Did you look into using this technique to resolve `..` in a way that respects following symlinks?

Reply:
On windows symlinks are resolved after resolving ".." (https://github.com/ziglang/zig/issues/7751)